### DISCUSSION TITLE
R4VS book

### DISCUSSION THREAD

**Discussion by rcatlord** [tech-acs/crvsreportpackage#55] [Announcements]
Hi @beardl-ons, @diego-ons, @gulilattesfaye, @jeroenminderman, @RachelS-ONS, and @tbelay. The R4VS book has a new home on [tech-acs](https://github.com/tech-acs/r4vs) and is published at [https://tech-acs.github.io/r4vs/](https://tech-acs.github.io/r4vs/). I will delete the version on my personal GitHub page.

The process for adding new content is very straightforward:

1. Create a `.qmd` file and include your markdown and r chunks
2. Add the file name to the chapters listed in `_quarto.yml`
3. Click **Render** to preview the results
4. Commit you changes and push to GitHub
5. Switch to the terminal and run `quarto publish gh-pages`

There is more guidance on the [quarto](https://quarto.org/docs/books/) website.

**Comment by tbelay** [comment]
Thank you @rcatlord. The content already included, especially on choropleth maps is comprehensive and easy to follow. The idea of having a companion book is valuable as it provides detailed explanations and a handy reference that complements the package. Thank you for your initiative.

**Reply by rcatlord** [reply]
You are very welcome.

**Reply by gulilattesfaye** [reply]
This is very helpful and well organized. Is there any way to download the book in one go.

**Reply by rcatlord** [reply]
Thanks Gulilat. Yes, you can create a downloadable PDF document. However, it is tricky to get the right LaTeX distribution for Quarto with firewall restrictions. I will give it a go on my personal laptop.
